enable-ius.sh

```diff
@@ -3,6 +3,12 @@
 # Script to setup the IUS public repository on your EL server.
 # Tested on CentOS/RHEL 6/7.
 
+set -o errexit
```

Comment:
As this is an alias for `set -e`, let's just use `set -e` directly instead.

Reply:
Done. I normally prefer `set -e` myself, but I thought being more verbose might be preferred for sample code.